manage.py

```diff
@@ -12,6 +14,11 @@
 if __name__ == "__main__":
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "project.settings")
 
+    # Override default port for `runserver` command
+    django.setup()
+    from django.core.management.commands.runserver import Command as runserver
+    runserver.default_port = "38000"
```

Comment:
This feels very hacky. Could you overwrite the runserver command with your own and then just subclass it?